packages/nns/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export type { DeployedSns } from "../candid/sns_wasm.idl";
+export type { DeployedSns } from "../../../candid/sns_wasm";
```

Comment:
is the sns_wasm the SNS Manager mentioned in other schemas?

Reply:
yes

it changed names a couple of times but is now (according the answer I got about it last week from nns team) named "sns-wasm"